core/actionProxy/actionproxy.py

```diff
@@ -127,18 +128,29 @@ def error(msg):
 
         try:
             input = json.dumps(args)
-            p = subprocess.Popen(
-                [self.binary, input],
-                stdout=subprocess.PIPE,
-                stderr=subprocess.PIPE,
-                env=env)
+            if len(input) > 131071:             # MAX_ARG_STRLEN (131071) linux/binfmts.h
+                # pass argument via stdin
```

Comment:
The if/else could be simpler.

Reply:
I agree and can also read your mind 😉 
Send the PR and I will merge